atm.js

```diff
@@ -1,21 +1,69 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready(function(){
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//Checking Deposit
+  $(".cdepositButton").on("click",function(){
+    console.log("i clicked the deposit button")
+    var deposit = parseInt($(".input").val())
+    //console.log(deposit)
+   var balance=$(".cBalance").text()
+   //console.log(balance)
+   balance= parseInt(balance.replace("$", ""))
+  var newBalance= deposit + balance
+ //console.log(balance)
+ $(".cBalance").text("$" + newBalance)
+ //I wasn't sure what instructions meant regarding changing the color to reflect the balance
+ if (newBalance >= 500){
+ $(".cBalance").css("background-color", "#6df15d")
+ }
 
-}
 
-function checkingWithdrawal(amount){
+ })
 
+//Checking Withdrawal
+$(".cwithdrawalButton").on("click",function(){
+  console.log("i clicked the withdrawal button")
+var withdrawal = parseInt($(".input").val())
+var balance=$(".cBalance").text()
+if (withdrawal > balance) { //unable to figure out the code to ignore transaction if withdrawal > deposit total 
+var balance=$(".cBalance").text()
 }
-
-function savingsDeposit(amount){
+balance= parseInt(balance.replace("$", ""))
+  var newBalance= balance - withdrawal
+   $(".cBalance").text("$" + newBalance)
+   if (newBalance === 0){
+   $(".cBalance").css("background-color", "#de3ea8")
 
 }
+})
 
-function savingsWithdrawal(amount){
-
+//Savings Deposit
+$(".sdepositButton").on("click",function(){
+  console.log("i clicked the deposit button")
+  var deposit = parseInt($(".sInput").val())
+  //console.log(deposit)
+ var balance=$(".sBalance").text()
+ //console.log(balance)
+ balance= parseInt(balance.replace("$", ""))
```

Comment:
Remember to indent code and fix whitespace - it makes code much easier to read, and consequently easier to debug!